modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -36,6 +54,20 @@ function h3ToPolygon(hexId) {
   return vertices;
 }
 
+function mergeTriggers(currentTrigger, newTrigger) {
```

Comment:
What's the point of creating a `newTrigger` object if you're just going to use `newTrigger.cover`?

Reply:
Either I have to pass value and do {coverage: value} or pass an object and access it as object.coverage. Changed it to latter former case.